build/yamls/antrea-eks.yml

```diff
@@ -1629,6 +1629,93 @@ spec:
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
+metadata:
```

Comment:
forgive my ignorance, but can this feature be leverage on the EKS dataplane (I seem to recall Antrea runs in policy only mode there)

Reply:
Considering this is autogenerated, I'm not sure if there is a way to make platform-specific changes?